code/datums/autolathe/autolathe_datums.dm

```diff
@@ -38,17 +38,21 @@ other types of metals and chemistry for reagents).
 
 //These procs are used in subtypes for assigning names and descriptions dynamically
 /datum/design/proc/AssembleDesignInfo()
+
```

Comment:
Possible unstated change?

Reply:
ah this was an attempt to fix a runtime error, but reverted, its got to be done later